test/functional/cmdLineTests/gptest/playlist.xml

```diff
@@ -51,5 +51,41 @@
 		<types>
 			<type>native</type>
 		</types>
+        <subsets>
+            <subset>8</subset>
+        </subsets>
```

Comment:
Could we make the `<subsets>` alined? Same as below.

Reply:
Alinements have been fixed. Thanks.